ceph-docker-lint/config/definitions/ceph-docker-lint.yml

```diff
@@ -0,0 +1,63 @@
+- scm:
+    name: ceph-docker
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-docker.git
+          branches:
+            - ${sha1}
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          wipe-workspace: false
+          basedir: "ceph-docker"
+
+- job:
+    name: ceph-docker-lint
+    node: small && centos7
+    defaults: global
+    display-name: 'ceph-docker-lint'
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph-docker/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - alfredodeza
+            - ktdreyer
+            - gmeno
+          white-list:
+            - xarses
+            - angdraug
+            - mbroz
+            - ddiss
+            - osynge
+          org-list:
+            - ceph
+          trigger-phrase: ''
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: false
+          auto-close-on-fail: false
+
+    scm:
+      - ceph-docker
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
```

Comment:
This doesn't hurt, but it doesn't look like you're using anything here from that file.